content/en/docs/Contribute/style-guide.md

```diff
@@ -0,0 +1,675 @@
+---
+title: Documentation style guide
+linktitle: Style guide
+weight: 40
+cSpell:ignore:
+slug: style-guide
+---
+
+<!-- overview -->
+This page gives writing style guidelines for the OpenTelemetry documentation.
+These are guidelines, not rules. Use your best judgment, and feel free to
+propose changes to this document in a pull request.
+
+For additional information on creating new content for the OpenTelemetry
+documentation, read the [Documentation Content Guide](/docs/contribute/style/content-guide/).
+
+Changes to the style guide are made by SIG Docs as a group. To propose a change
+or addition, [add it to the agenda](https://bit.ly/sig-docs-agenda) for an upcoming
+SIG Docs meeting, and attend the meeting to participate in the discussion.
+
+<!-- body -->
+
+{{< note >}}
+OpenTelemetry documentation uses
+[Goldmark Markdown Renderer](https://github.com/yuin/goldmark)
+with some adjustments along with a few
+[Hugo Shortcodes](/docs/contribute/style/hugo-shortcodes/) to support
+glossary entries, tabs, and representing feature state.
+{{< /note >}}
+
+## Language
+
+OpenTelemetry documentation has been translated into multiple languages
+(see [Localization READMEs](https://github.com/open-telemetry/opentelemetry.io/blob/main/README.md#localization-readmemds)).
+
+The way of localizing the docs for a different language is described in [Localizing OpenTelemetry Documentation](/docs/contribute/localization/).
+
+The English-language documentation uses U.S. English spelling and grammar.
+
+{{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
+
+## Documentation formatting standards
+
+### Use upper camel case for API objects
```

Comment:
This section might need to read something like "Use idiomatic case for API and SDK references"
There's several references in the spec that implementations should follow language idioms. 

Not sure if maybe references to the spec should be made at this point.